% Acronym Definitions
\setacronymstyle{long-short}

% Biology
\newacronym{DNA}{DNA}{deoxyribonucleic acid}
\newacronym{RNA}{RNA}{ribonucleic acid}
\newacronym{GEO}{GEO}{Gene Expression Omnibus}
\newacronym{NCBI}{NCBI}{National Center for Biotechnology Information}
\newacronym{SRA}{SRA}{Sequence Read Archive}
\newacronym{TAD}{TAD}{Topologically Associating Domain}
\newacronym{LAD}{LAD}{Lamina Associating Domain}
\newacronym{DS}{DS}{double-stranded}
\newacronym{GC}{GC}{guanine-cytosine}

\newacronym{PCA}{PCA}{Principal Component Analysis}
\newacronym{SVD}{SVD}{Singular Value Decomposition}
\newacronym{PC}{PC}{Principal Component}
\newacronym{ICE}{ICE}{Iterative Correction and Eigenvector Decomposition}
\newacronym{DI}{DI}{Directionality Index}
\newacronym{EM}{EM}{Expectation Maximization}
\newacronym{IPF}{IPF}{Iterative Proportional Fitting}
\newacronym{MLE}{MLE}{Maximum Likelihood Estimation}

% Glossary definitions
\newglossaryentry{karyotype}{%
  name={karyotype},
  description={A photomicrograph of chromosomes arranged according to a standard classification}%
}

\newglossaryentry{trans contacts}{%
  name={trans contacts},
  description={Interactions that occur between bins on two different chromosomes}%
}

\newglossaryentry{cis contacts}{%
  name={cis contacts},
  description={Interactions that occur between bins on the same chromosome}%
}

\newglossaryentry{contact map}{%
  name={contact map},
  description={A non-negative, square matrix recording observed interactions between different genomic regions}%
}

\newglossaryentry{polymer}{%
  name={polymer},
  description={A substance that has a molecular structure consisting chiefly or entirely of a large number of similar units bonded together, e.g., many synthetic organic materials used as plastics and resins}
}

\newglossaryentry{epigenetic}{%
  name={epigenetic},
  description={Any heritable influence on gene activity, unaccompanied by a change in the DNA}
}

\newglossaryentry{restriction enzyme}{%
  name={restriction enzyme},
  description={An enzyme that restricts, or performs double-strand cut, at a specific DNA sequence motif}
}

\newglossaryentry{ligation}{%
  name={ligation},
  description={The joining of two DNA strands or other molecules by a phosphate ester linkage}
}

\newglossaryentry{oligonucleotide}{%
  name={oligonucleotide},
  description={A polynucleotide whose molecules contain a relatively small number of nucleotides}
}

\newglossaryentry{nucleosome}{%
  name={nucleosome},
  description={A chromatin secondary structure consisting of $\sim147$ base pairs of DNA wrapped 1.75 times around an octamer of core histone proteins}
}

\newglossaryentry{scree plot}{%
  name={scree plot},
  description={A graph displaying the eigenvalue associated with each component in descending order versus the component number.  Used in PCA to determine how many components to consider in the analysis}
}

\newglossaryentry{eigenspectrum}{%
  name={eigenspectrum},
  description={A spectrum of eigenvalues}
}

\newglossaryentry{normalization}{%
  name={normalization},
  description={An attempt to compensate for noise and systematic bias from in a particular data set, while preserving the differences in a data set}
}

\newglossaryentry{toric model}{%
  name={toric},
  description={A background model for a distribution.  Also known as a log-linear model}
}

\newglossaryentry{log-linear model}{%
  name={log-linear model},
  description={See toric model.}
}

\newglossaryentry{transcriptome}{%
  name={transcriptome},
  description={The sum total of all messenger RNA molecules expressed from genes of an organism}
}

\newglossaryentry{contingency table}{%
  name={contingency table},
  description={A type of table in statistics that displays the frequency distribution of variables.  Also called a crosstab}
}

\newglossaryentry{sufficient statistic}{%
  name={sufficient statistic},
  description={A statistic that does as good a job estimating an unknown parameter as the entire sample}
}

\newglossaryentry{norm}{%
  name={norm},
  description={In linear algebra, a norm is a function that assigns a strictly non-negative length to each vector in a vector space}
}
