% Acronym Definitions
\setacronymstyle{long-short}

% Biology
\newacronym[description=any of various nucleic acids that are usually the molecular basis of heredity]
{DNA}{DNA}{deoxyribonucleic acid}

\newacronym[description=any of various nucleic acids that are usually the molecular basis of heredity]
{bp}{bp}{base pair}

\newacronym[description=any of various nucleic acids that contain ribose and uracil as structural components and are associated with the control of cellular chemical activities—called also ribonucleic acid]
{RNA}{RNA}{ribonucleic acid}

\newacronym[description=the Gene Expression Omnibus.  Located at \url{http://www.ncbi.nlm.nih.gov/geo/}]{GEO}{GEO}{Gene Expression Omnibus}

\newacronym[description=the National Center for Biotechnology Information.  Located at \url{http://www.ncbi.nlm.nih.gov/}]
{NCBI}{NCBI}{National Center for Biotechnology Information}

\newacronym[description=stores raw sequencing data and alignment information from high-throughput sequencing platforms.  For more information see \url{http://www.ncbi.nlm.nih.gov/sra}]
{SRA}{SRA}{Sequence Read Archive}

\newacronym[description=local regions of highly enriched chromatin interactions]
{TAD}{TAD}{Topologically Associating Domain}

\newacronym[description=genome-lamina interacting domains found to be 0.1--10 megabases in size]
{LAD}{LAD}{Lamina Associated Domain}

\newacronym{DS}{DS}{double-stranded}

\newacronym{GC}{GC}{guanine-cytosine}

\newacronym[description=human embryonic stem cell line cultured in lab]
{hESC}{hESC}{Human embryonic stem cell}

\newacronym[description=the location where transcription starts at the 5'-end of a gene sequence]
{TSS}{TSS}{Transcriptional Start Site}

\newacronym[
  \glsshortpluralkey={GTFs},
  description=a class of protein transcription factors that bind to specific sites (promoter) on DNA to activate transcription of genetic information from DNA to messenger RNA
]{GTF}{GTF}{General Transcription Factor}

\newacronym{PIC}{PIC}{preinitiation complex}

\newacronym[longplural={chromosome territories}]{CT}{CT}{chromosome territory}

\newacronym{3C}{3C}{Chromosome Conformation Capture}

\newacronym{4C}{4C}{Chromosome Conformation Capture-on-Chip}

\newacronym{5C}{5C}{Chromosome Conformation Capture Carbon-Copy}

\newacronym{NGS}{NGS}{Next Generation Sequencing}

\newacronym{COSMIC}{COSMIC}{The Catalogue of Somatic Mutations in Cancer}

\newacronym[description=a specific heritable point on a chromosome that tends to form a gap or constriction and may tend to break when the cell is exposed to partial replication stress]
{CFS}{CFS}{chromosome fragile site}


% maths
\newacronym{PCA}{PCA}{Principal Component Analysis}
\newacronym{SVD}{SVD}{Singular Value Decomposition}
\newacronym{PC}{PC}{Principal Component}
\newacronym{ICE}{ICE}{Iterative Correction and Eigenvector Decomposition}
\newacronym{DI}{DI}{Directionality Index}
\newacronym{EM}{EM}{Expectation Maximization}
\newacronym{IPF}{IPF}{Iterative Proportional Fitting}
\newacronym{MLE}{MLE}{Maximum Likelihood Estimation}

% Glossary definitions
\newglossaryentry{karyotype}{%
  name={karyotype},
  description={a photomicrograph of chromosomes arranged according to a standard classification}%
}

\newglossaryentry{trans contact}{%
  name={\textit{trans} contact},
  plural={\textit{trans} contacts},
  description={interactions that occur between bins on two different chromosomes}%
}

\newglossaryentry{cis contact}{%
  name={\textit{cis} contact},
  plural={\textit{cis} contacts},
  description={interactions that occur between bins on the same chromosome}%
}

\newglossaryentry{contact map}{%
  name={contact map},
  description={a non-negative, square matrix recording observed interactions between different genomic regions}%
}

\newglossaryentry{nucleobase}{%
  name={nucleobase},
  description={nitrogen-containing biological compounds (nitrogenous bases) found linked to a sugar within nucleosides—the basic building blocks of deoxyribonucleic acid (DNA) and ribonucleic acid (RNA). B substance that has a molecular structure consisting chiefly or entirely of a large number of similar units bonded together, e.g., many synthetic organic materials used as plastics and resins}
}

\newglossaryentry{polymer}{%
  name={Polymer},
  description={a substance that has a molecular structure consisting chiefly or entirely of a large number of similar units bonded together, e.g., many synthetic organic materials used as plastics and resins}
}

\newglossaryentry{epigenetic}{%
  name={epigenetic},
  description={any heritable influence on gene activity, unaccompanied by a change in the DNA}
}

\newglossaryentry{restriction enzyme}{%
  name={restriction enzyme},
  description={an enzyme that restricts, or performs double-strand cut, at a specific DNA sequence motif}
}

\newglossaryentry{ligation}{%
  name={ligation},
  description={the joining of two DNA strands or other molecules by a phosphate ester linkage}
}

\newglossaryentry{nucleotide}{%
  name={nucleotide},
  description={a compound consisting of a nucleoside linked to a phosphate group. Nucleotides form the basic structural unit of nucleic acids such as DNA}
}

\newglossaryentry{oligonucleotide}{%
  name={oligonucleotide},
  description={a polynucleotide whose molecules contain a relatively small number of nucleotides}
}

\newglossaryentry{nucleosome}{%
  name={nucleosome},
  description={a chromatin secondary structure consisting of $\sim147$ base pairs of DNA wrapped 1.75 times around an octamer of core histone proteins}
}

\newglossaryentry{scree plot}{%
  name={scree plot},
  description={a graph displaying the eigenvalue associated with each component in descending order versus the component number.  Used in PCA to determine how many components to consider in the analysis}
}

\newglossaryentry{eigenspectrum}{%
  name={eigenspectrum},
  description={a spectrum of eigenvalues}
}

\newglossaryentry{normalization}{%
  name={normalization},
  description={an attempt to compensate for noise and systematic bias from in a particular data set, while preserving the differences in a data set}
}

\newglossaryentry{toric model}{%
  name={toric model},
  description={a background model for a distribution.  Also known as a log-linear model}
}

\newglossaryentry{log-linear model}{%
  name={log-linear model},
  description={see toric model}
}

\newglossaryentry{transcriptome}{%
  name={transcriptome},
  description={the sum total of all messenger RNA molecules expressed from genes of an organism}
}

\newglossaryentry{contingency table}{%
  name={contingency table},
  description={a type of table in statistics that displays the frequency distribution of variables.  Also called a crosstab}
}

\newglossaryentry{sufficient statistic}{%
  name={sufficient statistic},
  description={a statistic that does as good a job estimating an unknown parameter as the entire sample}
}

\newglossaryentry{norm}{%
  name={norm},
  description={in linear algebra, functional analysis and related areas of mathematics, a function that assigns a strictly positive length or size to each vector in a vector space—save possibly for a single zero vector, with length zero}
}

\newglossaryentry{X-inactivation}{%
  name={x-inactivation},
  description={a process by which one of the two copies of the X chromosome present in female mammals is inactivated}
}

\newglossaryentry{in vivo}{%
  name={in vivo},
  description={performed or taking place in a living organism}
}

\newglossaryentry{nucleosome array}{%
  name={nucleosome array},
  description={the fundamental building block of chromosomal superstructures, the substrate for transcription, and the first nucleoprotein assembly laid down after DNA replication}
}

\newglossaryentry{euchromatin}{%
  name={euchromatin},
  plural={euchromatin},
  description={a lightly packed form of chromatin (DNA, RNA and protein) that is rich in gene concentration, and is often (but not always) under active transcription}
}

\newglossaryentry{heterochromatin}{%
  name={heterochromatin},
  plural={heterochromatin},
  description={chromosome material of different density from normal (usually greater), in which the activity of the genes is modified or suppressed}
}

\newglossaryentry{preinitiation complex}{%
  name={preinitiation complex},
  plural={preinitiation complexes},
  description={large complex of promoter-bound proteins that is necessary for the transcription of protein-coding genes in eukaryotes}
}

\newglossaryentry{high-throughput}{%
  name={high-throughput},
  description={the use of automation equipment with classical cell biology techniques to address biological questions that are otherwise unattainable using conventional methods}
}

\newglossaryentry{fragile site}{%
  name={fragile site},
  description={heritable-specific loci on human chromosomes that exhibit nonrandom gaps or breaks when chromosomes are exposed to specific cell culture conditions}
}

\newglossaryentry{core promoter}{%
  name={core promoter},
  description={the minimal portion of the promoter required to properly initiate gene transcription}
}

\newglossaryentry{variance}{%
  name={variance},
  description={measurement of how far a set of numbers is spread out}
}
