\chapter{Methods}

\section*{Data Collection}

We selected IMR90 lung fibroblast and H1 \glsentryfull{hESC} cell lines for analysis due to the availability of data and
preexisting literature on these cells lines.

\subsection*{Processing Chromatin Interaction Data}

We obtained Hi-C data sets from the Gene Expression Omnibus (GEO)\cite{edgar2002} as \gls{SRA} files,
accession number \href{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE43070}{GSE43070}.  Chimeric reads were realigned
to the human genome (hg19) using the BowTie2 alignment algorithm on 16 threads\cite{langmead2012}.  The resulting mapped
reads were stored as HDF5 heat maps on disk for later interrogation.


\subsection*{Iterative correction}
Using the Hi-C library developed by Imakaev and colleagues\cite{imakaev2012}, the heatmaps were iteratively corrected
and decomposed into principal components.

% FIXME Make sure appendix numbers/labels are correct
We noted that the aligned interactions matrices by replicate are highly correlated (Spearman's $\rho \geq 0.69$)  % FIXME p-value!
for both cell lines, and significantly different from random (Appendix III).  The

% TODO Do the python statistics here to validate that the data is good.  This includes:
% 1) Spearman's between H1 and IMR90
% 2) Spearman's between H1 replicates
% 3) Compute a "random" Hi-C map and take correlation between these
% 4) Show that normalization does not significantly decrease the correlation
% 5) Include p-values for everything I state

\subsection*{Probe Scaling}




\subsection*{Gene Expression and Pathway Analysis}

Affymetrix gene expression microarray data for IMR90 and H1 hESCs were obtained from GEO (GSE2672\cite{kim2005}
and GSE54186\cite{kim2014}, respectively).  Probe annotations were downloaded from the Affymetrix site for
the Human Genome U133v2 array, and remapped to genes from UCSC\@.  Noticing that expression profiles for replicates
were similar, the gene expression levels for each cell line were calculated by averaging across replicates, and
taking the log2 transform of the averaged signal, yielding signal for $23,100$ genes.

% TODO Kolmogorov-Smirnov test w/ normal distribution

To understand expression changes during differentiation from stem cell to lung cell, I subtracted the lung expression
signal from the stem cell line.  Taking the log2 transform yields the fold change in gene expression level per gene.
Two groups of genes were selected for further analysis: the 100 most upregulated genes and the 100 most downregulated
genes.  I was curious to find out whether classes of genes were changing together.  Using the ConsensusPathDB
tool\cite{kamburov2012}, I performed over-representation analysis for pathways and gene ontologies.

