% An Applied Mathematic/Biology Thesis
% By Jonathan Niles

\documentclass[phd,tocprelim]{cornell}

% packages
\usepackage{graphicx,pstricks}
\usepackage{graphics}
\usepackage{moreverb}
\usepackage{subfigure}
\usepackage{epsfig}
\usepackage{subfigure}
\usepackage{hangcaption}
\usepackage{txfonts}
\usepackage{palatino}
\usepackage[toc]{glossaries}

% Set tolerance for box model
\tolerance=9999


\renewcommand{\caption}[1]{\singlespacing\hangcaption{#1}\normalspacing}
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}
\renewcommand{\floatpagefraction}{0.75}

\title{Structural Machinery for Chromosomal Fragility and Translocation Frequency}
\author{Jonathan Niles}
\conferraldate{June}{2014}
\degreefield{Bachelor of Science}
\copyrightholder{Jonathan Niles}
\copyrightyear{2014}

\bibliographystyle{cbe}

% Glossary Definitions
\newacronym{dna}{DNA}{Deoxyribonucleic Acid}
\newacronym{rna}{RNA}{Ribonucleic Acid}
\newacronym{geo}{GEO}{Gene Expression Omnibus}
\newacronym{ncbi}{NCBI}{National Center for Biotechnology Information}
\newacronym{sra}{SRA}{Sequence Read Archive}
\newacronym[longpural=Topologically Association Domains]
  {tad}{TAD}{Topologically Associating Domain}
\newacronym[longpural=Lamina Association Domains]
  {lad}{LAD}{Lamina Associating Domain}

% Make a glossary using the glossaries package
\makeglossaries

\begin{document}

\maketitle
\makecopyright

\begin{abstract}
Chromosomal rearrangements and sequence translocations are
canonical hallmarks of cancerous cells.  The canonical models
for these chromosomal breakage events involve spatially
independent molecular mechanisms of double-strand break
repair and non-homologous end joining.  The recent development of high
throughput techniques to interrogate genome-wide chromatin
conformation allows for the spatial dynamics of chromosomal
breakage to be examined and correlated with translocations
frequencies and other genomic rearrangements.  I present
evidence for translocaction frequency based on distances between
translocating sites in 3D space.  I also propose experiments
to further elucidate the interplay between distance and
genomic rearrangements in cells.
\end{abstract}

\begin{dedication}
This document is dedicated to Miriam Paul Fountain.
\end{dedication}

\begin{acknowledgements}
Dr. Tyrone Ryba
Dr. Patrick McDonald
Dr. David Gillman
\end{acknowledgements}

\contentspage
\tablelistpage
\figurelistpage

\normalspacing \setcounter{page}{1} \pagenumbering{arabic}
\pagestyle{cornell} \addtolength{\parskip}{0.5\baselineskip}

% make the glossaries
\makeglossaries

\chapter{Introduction}

The human genome is a three billion nucleotide string that encodes the
the vast instruction set for the cells growth, function, and death.  The cell
programme, however, is dictated not only by the instruction set, but also
how the instructions are read, compiled, and exectuted, and is influenced
by a myriad of factors in addition to the primary sequence.  Increasingly,
modern cancer research is indicating this epigenetic landscape plays strategic
role in determining cell fate.  Understanding the landmark epigenetic features,
the conditions which bring about their development, and the impact they have on
the cell is of paramount importance and difficultly.

Perhaps the most common and least understood effects of genomic instability are
translocations between disperate genomic regions.  Recently, several proposed
molecular mechanisms have gained traction in explaining how translocations arise
but why they arise is a question still without an answer.

Here, I investigate genome-wide interaction data to and show that proximity in
genomic space strongly correlates with translocation probability.


\chapter{Methods}

\section{Data Collection}

The HiC datasets used in this thesis were downloaded from the Gene
Expression Omnibus as Sequence Read archive files (.sra).
  \footnote{
    The data can be acquire under access number
    \href{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE43070}
    {GSE43070}.\cite{Ren2013}
  }
A lung fibroblast cell line, IMR90 was chosen due to the abundance of
HiC interaction data, as well as being a represenatative cell line
for lung cancer studies.  A script was written to recover six experimental
replicates from GEO.

The cell lines were prepared with no treatment according to the protocol
described by Dekker and collegues\cite{Dekker2013}.  In brief, the cells
are washed with a formaldehyde fixation solution to cross-links
DNA and protein molecules near each other in genomic space.  The cells are
lysed and the nuclear contents are treated with a non-specific restriction
enzyme, \textit{HindIII}.  After restriction, the remaining nuclear material
is washed so that the remaining contents are short DNA sequences or cross
-linked DNA-protein complexes.  The DNA reads are labeled with a biotin
marker and ligated at low concentrations that favor within ligation between
reads bound to the same complex.  The reads are captured and sequenced,
building what is known as a HiC library.

%% Regulatory Elements from Ensembl

%% Translocations from COSMIC

%% Genes from Biomart
%% Hg19 from UCSC


\section{Mapping and Alignment}

A HiC library is formed largely of \texit{chimeric} DNA sequences,
sequences composed of material from more than one source.  The ligation process
creates novel DNA sequences with a portion of sequence from one region of
the genome and the remaining portion from a different genomic region.  In
order to discover to which two regions each probe corresponds, the probes
must be realigned to the human genome.  Previous methods (\textit{citation needed})
have truncated reads to a fixed cutoff size before realignment with a short
read alignment algorithm.  In a recent paper, Imakaev and collegues
describe an algorithm to repeated algin reads with different truncation sizes
and analyzing reads with the maximum alignment score\cite{Imakaev2012}.  We followed
this proceedure to realign the IMR90 HiC libraries for six replicates to
the Human Genome build 19 (hg1), with an average of XX million reads
mapped per replicate.

%% Data Validation

\section{Analysis}

%% Iterative Correction and Eigenvalue Expansion

%% Visualization Tools
%% Replication Timing Analysis..?


%% The Graph Theory Section
\chapter{The Interaction Graph}

\section{Motivation and Preliminaries}

The interation matrix $M$ describes an undirected graph of genomic interactions
at the selected bin size.  It is natural to leverage graph theoretic ideas
and extend our analysis to the topology and structure of the interaction graph,
to better understand the underlying chromatin structure.  Additionally, graph
theory contains strong notions of network stability and degree, allowing us an
additional avenue to create comparisons between experiments, cell lines, and cell
states.  Finally, graphs are a powerfully illustrative tool, allowing the
researcher to make quick judgements about the dataset and the generating
mechanics which produced it.

\begin{definition}
  Let $V_{n}$ be a collection of $n$ genomic bins (vertices).  Let $E_{m}$ be a
  collection of $m$ interactions (edges) between bins in $V_{n}$ with weights
  $w_{m}$ corresponding to interaction frequencies.  We define the  genomic
  interaction graph at resolution $n$ to be $G_{n} := (V_{n}, E_{n})$.
\end{definition}

The graph $G_{n}$ is an undirected graph that decribes the all-to-all
interactions from a given Hi-C experiment.  Since the human genome is
partitioned in the nucleus into chromosomal territories, it makes sense to
partition the graph by chromosome to better reflect the biological reality.

\begin{definition}
  Let $C_{n,i}$ be a partition of $G_{n}$ where all bins in $C_{n,i}$ are
  located in chromosome $i$.
\end{definition}

The graph partition $C_{n,i}$ gives us a tool to analyze the topology of
intrachrosomal interations.

\section{Analysis}

Now we have developed the terminology and intution into the interaction
graphs below, we can begin to interrogate and evaluation our model.

\subsection{Cycles}

It is of interest to determine if there are any cycles present in the
interaction graphs.  Intuitvely, cycles correspond to molecular domains which
may have functional significance.  Our approach is to identify cycles at the
smallest resolution possible, and gradually increase our bin size to understand
how these small regulatory units fit into the megabase-size and larger regulatory
landscape.  It is also of interest to determine if cycles cease to exist in the
graph after relaxing past certain resolutions, perhaps characterising chromosomes
that are composed of many smaller, independent domains rather than large loops.



\chapter{Results & Discussion}


%
% APPENDIX
%

\appendix
\appendixpage
\addappheadtotoc
\chapter{Iterative Alignment of Probes}

Probes were aligned to the human genome build 19 using proceedures outlined by
Imakaev and collegues\cite{imakaev2013}.  The chimeric nature of the reads
requires that probes be aligned iteratively, starting from a small, truncated
region from the beginning of the read, mapping this truncated area, increasing
the truncation size and recursing a fixed number of steps or until the alignment
scores become sufficiently poor.  Due to the large number of reads requiring
alignment, we opted to use a fixed truncation length (based on sequence length)
and four steps in the iterative alignment protocol.  The calculation
for the truncation and step size can be found in the the iterativeMapping.py
script provided in the Appendix: Code.  Most reads were 100 base pairs, resulting
in an initial truncation length of 28 base pairs, and step size of 18 base pairs.

Using the mapping functionality from the hiclib python package\cite{imakaev2013},
sequences from the six experimental replicates were realigned to the genome.  The
alignment employed the fast Bowtie2 alignment algorithm\cite{Langmead2012}.  Once
aligned, the probes were stored as an interaction matrix in the high performance
HDF5\cite{hdf5} data format, a total of 25Gb for all replicates.


\chapter{Spearman's Rank Correlation Calculation}

In order to make meaningful comparisons between datasets (replicates,
in this case), we must show that some degree of relationship exists between
the datasets and comparisons or combinations of the data from disperate sets
are valid to a degree of uncertainty.  It is also essential to understand if the
experimental replicates indeed managed to replicate the conditions of the primary
experiment, or if experimental errors prevent the comparison between replicates.

Spearman's Rank Correlation Coefficient (denoted by the Greek letter \textit{rho}
$\rho$) is a nonparametric measurei of association between two variables.
Spearman's coefficient assumes some monotonic relationship between variables,
rather than a linear relationship (as in Pearson's), making it appropriate
to compare the IM90 interaction datasets.  The formula for Spearman's $\rho$ is
given as follows:

$
\rho = 1 - \frac{\sum_{i=1}{n}(d_i^2)}{n(n^2 - 1)}
$

where $\rho$ is the correlation coefficient taking values between $-1$ and $+1$,
$d_i = x_i - y_i$ where $x_i, y_i$ are ranks derived from the raw scores $X$ and
$Y$ respectively.

%% Question: Is this symmetric? To some degree?

The first replicate IMR90 interation dataset was labeled the "primary" dataset
and the remaining five were compared using Spearman's Rank Correlation.  The
results are given in Table X.

%% Example data for now.
\begin{tabular}{l c}
  \hline
  Replicate Number & $\rho$ \\
  \hline
  2 && 0.34 \\
  3 && 0.76 \\
  4 && 0.94 \\
  5 && 0.81 \\
  6 && 0.49 \\
  \hline
  \caption{Spearman's $\rho$ for all experimental replicates.}
  \label{tab:spearmans}
\end{tablular}

%
% BIBLIOGRAPHY
%

\bibliography{bibliography}

\end{document}
