\documentclass[phd,tocprelim]{cornell}

% An Applied Mathematics/Biology thesis by Jonathan Niles

%Some possible packages to include
\usepackage{graphicx,pstricks}
\usepackage{graphics}
\usepackage{moreverb}
\usepackage{subfigure}
\usepackage{epsfig}
\usepackage{subfigure}
\usepackage{hangcaption}
\usepackage{txfonts}
\usepackage{palatino}

%if you're having problems with overfull boxes, you may need to increase
%the tolerance to 9999
\tolerance=9999

\bibliographystyle{cbe}

\renewcommand{\caption}[1]{\singlespacing\hangcaption{#1}\normalspacing}
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}
\renewcommand{\floatpagefraction}{0.75}

\title {Structural Machinery for Chromosomal Fragility and Translocation Frequency}
\author {Jonathan Niles}
\conferraldate {June}{2014}
\degreefield {Bachelor of Science}
\copyrightholder{Jonathan Niles}
\copyrightyear{2014}

\begin{document}

\maketitle
\makecopyright

\begin{abstract}
I present evidence that the 3D structure of the genome regulates
chromosomal fragility, heavily determining the translocation
frequencies between neighboring sequences and providing insight
into the mechanism between chromosomal fragile site breakage.
\end{abstract}

\begin{biosketch}
Your biosketch goes here. Make sure it sits inside
the brackets.
\end{biosketch}

\begin{dedication}
This document is dedicated to Miriam Fountain.
\end{dedication}

\begin{acknowledgements}
Dr. Tyrone Ryba
Dr. Patrick McDonald
Dr. David Gillman
\end{acknowledgements}

\contentspage
\tablelistpage
\figurelistpage

\normalspacing \setcounter{page}{1} \pagenumbering{arabic}
\pagestyle{cornell} \addtolength{\parskip}{0.5\baselineskip}

\chapter{Introduction}

%% \section{SECTION 1}
%% The text for Section 1 goes here, without brackets.
%%
%% \section{SECTION 2}
%% Section 2 text.
%%
%% \subsection{Subsection heading goes here}
%%
%% Subsection 1 text
%%
%% \subsubsection{Subsubsection 1 heading goes here}
%% Subsubsection 1 text
%%
%% \subsubsection{Subsubsection 2 heading goes here}
%% Subsubsection 2 text
%%
%% \section{SECTION 3}
%% Section 3 text. The dielectric constant at the air-metal interface
%% determines the resonance shift as absorption or capture occurs.
%%
%% \begin{equation}
%% k_1=\frac{\omega }{c({1/\varepsilon_m + 1/\varepsilon_i})^{1/2}}=k_2=\frac{\omega
%% sin(\theta)\varepsilon_{air}^{1/2}}{c}
%% \end{equation}
%%
%% \noindent
%% where $\omega$ is the frequency of the plasmon, $c$ is the speed of
%% light, $\varepsilon_m$ is the dielectric constant of the metal,
%% $\varepsilon_i$ is the dielectric constant of neighboring insulator,
%% and $\varepsilon_{air}$ is the dielectric constant of air.

\chapter{Methods}

\section{Data Collection \& Processing}

The HiC datasets used in this thesis were downloaded from the Gene
Expression Omnibus as Sequence Read archive files (.sra).
  \footnote{
    The data can be acquire under access number
    \href{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE43070}
    {GSE43070}.\cite{Ren2013}
  }
A lung fibroblast cell line, IMR90 was chosen due to the abundance of
HiC interaction data, as well as being a represenatative cell line
for lung cancer studies.  A script was written to recover six experimental
replicates from GEO.

The cell lines were prepared with no treatment according to the protocol
described by Dekker and collegues\cite{Dekker2013}.  In brief, the cells
are washed with a formaldehyde fixation solution to cross-links
DNA and protein molecules near each other in genomic space.  The cells are
lysed and the nuclear contents are treated with a non-specific restriction
enzyme, \textit{HindIII}.  After restriction, the remaining nuclear material
is washed so that the remaining contents are short DNA sequences or cross
-linked DNA-protein complexes.  The DNA reads are labeled with a biotin
marker and ligated at low concentrations that favor within ligation between
reads bound to the same complex.  The reads are captured and sequenced,
building what is known as a HiC library.

The HiC library is formed largely of \texit{chimeric} DNA sequences, or
sequences made of material from more than one source.  The ligation process
creates novel DNA sequences with a portion of sequence from one region of
the genome and the remaining portion from a different genomic region.  In
order to discover to which two regions each probe corresponds, the probes
must be realigned to the human genome.  Previous methods (\textit{citation needed})
have truncated reads to a fixed cutoff size before realignment with a short
read alignment algorithm.  In a recent paper, Imakaev and collegues
describe an algorithm to repeated algin reads with different truncation sizes
and analyzing reads with the maximum alignment score\cite{Imakaev2012}.  We followed
this proceedure to realign the IMR90 HiC libraries for six replicates,
with an average of XX million reads remaining per replicate.



%% \chapter{Chapter 3}
%%
%% \chapter{Chapter 4}
%%
%%\appendix
%%\chapter{Chapter 1 of appendix}
%%Appendix chapter 1 text goes here
%%
%%\bibliography{bibliography}

\end{document}
