\documentclass[phd,tocprelim]{cornell}

% An Applied Mathematics/Biology thesis by Jonathan Niles

%Some possible packages to include
\usepackage{graphicx,pstricks}
\usepackage{graphics}
\usepackage{moreverb}
\usepackage{subfigure}
\usepackage{epsfig}
\usepackage{subfigure}
\usepackage{hangcaption}
\usepackage{txfonts}
\usepackage{palatino}

%if you're having problems with overfull boxes, you may need to increase
%the tolerance to 9999
\tolerance=9999

\bibliographystyle{cbe}

\renewcommand{\caption}[1]{\singlespacing\hangcaption{#1}\normalspacing}
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}
\renewcommand{\floatpagefraction}{0.75}

\title {Structural Machinery for Chromosomal Fragility and Translocation Frequency}
\author {Jonathan Niles}
\conferraldate {June}{2014}
\degreefield {Bachelor of Science}
\copyrightholder{Jonathan Niles}
\copyrightyear{2014}

\begin{document}

\maketitle
\makecopyright

\begin{abstract}
Chromosomal rearrangements and sequence translocations are
canonical hallmarks of cancerous cells.  The canonical models
for these chromosomal breakage events involve spatially
independent molecular mechanisms of double-strand break
repair and non-homologous end joining.  The recent development of high
throughput techniques to interrogate genome-wide chromatin
conformation allows for the spatial dynamics of chromosomal
breakage to be examined and correlated with translocations
frequencies and other genomic rearrangements.  I present
evidence for translocaction frequency based on distances between
translocating sites in 3D space.  I also propose experiments
to further elucidate the interplay between distance and
genomic rearrangements in cells.
\end{abstract}

\begin{dedication}
This document is dedicated to Miriam Paul Fountain.
\end{dedication}

\begin{acknowledgements}
Dr. Tyrone Ryba
Dr. Patrick McDonald
Dr. David Gillman
\end{acknowledgements}

\contentspage
\tablelistpage
\figurelistpage

\normalspacing \setcounter{page}{1} \pagenumbering{arabic}
\pagestyle{cornell} \addtolength{\parskip}{0.5\baselineskip}

\chapter{Introduction}

\chapter{Methods}

\section{Data Collection}

The HiC datasets used in this thesis were downloaded from the Gene
Expression Omnibus as Sequence Read archive files (.sra).
  \footnote{
    The data can be acquire under access number
    \href{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE43070}
    {GSE43070}.\cite{Ren2013}
  }
A lung fibroblast cell line, IMR90 was chosen due to the abundance of
HiC interaction data, as well as being a represenatative cell line
for lung cancer studies.  A script was written to recover six experimental
replicates from GEO.

The cell lines were prepared with no treatment according to the protocol
described by Dekker and collegues\cite{Dekker2013}.  In brief, the cells
are washed with a formaldehyde fixation solution to cross-links
DNA and protein molecules near each other in genomic space.  The cells are
lysed and the nuclear contents are treated with a non-specific restriction
enzyme, \textit{HindIII}.  After restriction, the remaining nuclear material
is washed so that the remaining contents are short DNA sequences or cross
-linked DNA-protein complexes.  The DNA reads are labeled with a biotin
marker and ligated at low concentrations that favor within ligation between
reads bound to the same complex.  The reads are captured and sequenced,
building what is known as a HiC library.

%% Regulatory Elements from Ensembl

%% Translocations from COSMIC

%% Genes from Biomart
%% Hg19 from UCSC


\section{Mapping and Alignment}

The HiC library is formed largely of \texit{chimeric} DNA sequences, or
sequences made of material from more than one source.  The ligation process
creates novel DNA sequences with a portion of sequence from one region of
the genome and the remaining portion from a different genomic region.  In
order to discover to which two regions each probe corresponds, the probes
must be realigned to the human genome.  Previous methods (\textit{citation needed})
have truncated reads to a fixed cutoff size before realignment with a short
read alignment algorithm.  In a recent paper, Imakaev and collegues
describe an algorithm to repeated algin reads with different truncation sizes
and analyzing reads with the maximum alignment score\cite{Imakaev2012}.  We followed
this proceedure to realign the IMR90 HiC libraries for six replicates to
the Human Genome build 19 (hg1), with an average of XX million reads
mapped per replicate.

%% Data Validation

\section{Analysis}

%% Iterative Correction and Eigenvalue Expansion

%% Visualization Tools
%% Replication Timing Analysis..?


%% The Graph Theory Section
\chapter{The Interaction Graph}

\section{Motivation and Preliminaries}

The interation matrix $M$ describes an undirected graph of genomic interactions
at the selected bin size.  It is natural to leverage graph theoretic ideas 
and extend our analysis to the topology and structure of the interaction graph,
to better understand the underlying chromatin structure.  Additionally, graph
theory contains strong notions of network stability and degree, allowing us an
additional avenue to create comparisons between experiments, cell lines, and cell
states.  Finally, graphs are a powerfully illustrative tool, allowing the
researcher to make quick judgements about the dataset and the generating
mechanics which produced it.

\begin{definition}
  Let $V_{n}$ be a collection of $n$ genomic bins (vertices).  Let $E_{m}$ be a
  collection of $m$ interactions (edges) between bins in $V_{n}$ with weights
  $w_{m}$ corresponding to interaction frequencies.  We define the  genomic
  interaction graph at resolution $n$ to be $G_{n} := (V_{n}, E_{n})$.
\end{definition}

The graph $G_{n}$ is an undirected graph that decribes the all-to-all
interactions from a given Hi-C experiment.  Since the human genome is
partitioned in the nucleus into chromosomal territories, it makes sense to
partition the graph by chromosome to better reflect the biological reality.

\begin{definition}
  Let $C_{n,i}$ be a partition of $G_{n}$ where all bins in $C_{n,i}$ are
  located in chromosome $i$.
\end{definition}

The graph partition $C_{n,i}$ gives us a tool to analyze the topology of
intrachrosomal interations.

\section{Analysis}

Now we have developed the terminology and intution into the interaction
graphs below, we can begin to interrogate and evaluation our model.

\subsection{Cycles}

It is of interest to determine if there are any cycles present in the
interaction graphs.  Intuitvely, cycles correspond to molecular domains which
may have functional significance.  Our approach is to identify cycles at the
smallest resolution possible, and gradually increase our bin size to understand
how these small regulatory units fit into the megabase-size and larger regulatory
landscape.  It is also of interest to determine if cycles cease to exist in the
graph after relaxing past certain resolutions, perhaps characterising chromosomes
that are composed of many smaller, independent domains rather than large loops.



\chapter{Results & Discussion}


%%\appendix
%%\chapter{Chapter 1 of appendix}
%%Appendix chapter 1 text goes here

\bibliography{bibliography}

\end{document}
